scheduler/classes/Member.py

```diff
@@ -0,0 +1,8 @@
+class Member:
```

Comment:
I REALLY like how you made a member and then base classes off of that. I think that clearly shows the abstraction we are going for. I think that's a perfect way of doing this for member, driver, rider